machiavelli-cards/src/main/java/pl/jellytech/machiavelli/cards/dtos/ExceptionResponse.java

```diff
@@ -0,0 +1,17 @@
+package pl.jellytech.machiavelli.cards.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+import java.sql.Timestamp;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class ExceptionResponse {
```

Comment:
do we need this? I'd prefer to have a Card object with an error inside

Reply:
I can extend dtos by some base class which will have error. But there is different responsibility between card and exception. Why do you prefer it?